# AmalgamGameTelemetry Index


## Operations
| Endpoint | Method | ID | Class | Wrapper |
|---|---|---|---|---|
| /game-telemetry/v1/protected/events | POST | protected_save_events_game_telemetry_v1_protected_events_post | [ProtectedSaveEventsGameTelemetryV1ProtectedEventsPost](../accelbyte_py_sdk/api/amalgam_game_telemetry/operations/operations/protected_save_events_game_telemetry_v1_protected_events_post.py) | [protected_save_events_game_telemetry_v1_protected_events_post](../accelbyte_py_sdk/api/amalgam_game_telemetry/wrappers/_operations.py) |
| /game-telemetry/v1/protected/steamIds/{steamId}/playtime | GET | protected_get_playtime_game_telemetry_v1_protected_steamIds__steamId__playtime_get | [ProtectedGetPlaytimeGameTelemetryV1ProtectedSteamIdsSteamIdPlaytimeGet](../accelbyte_py_sdk/api/amalgam_game_telemetry/operations/operations/protected_get_playtime_game_telemetry_v1_protected_steam_ids_steam_id_playtime_get.py) | [protected_get_playtime_game_telemetry_v1_protected_steam_ids_steam_id_playtime_get](../accelbyte_py_sdk/api/amalgam_game_telemetry/wrappers/_operations.py) |
| /game-telemetry/v1/protected/steamIds/{steamId}/playtime/{playtime} | PUT | protected_update_playtime_game_telemetry_v1_protected_steamIds__steamId__playtime__playtime__put | [ProtectedUpdatePlaytimeGameTelemetryV1ProtectedSteamIdsSteamIdPlaytimePlaytimePut](../accelbyte_py_sdk/api/amalgam_game_telemetry/operations/operations/protected_update_playtime_game_telemetry_v1_protected_steam_ids_steam_id_playtime_playtime_put.py) | [protected_update_playtime_game_telemetry_v1_protected_steam_ids_steam_id_playtime_playtime_put](../accelbyte_py_sdk/api/amalgam_game_telemetry/wrappers/_operations.py) |


## Models
| Model | Class |
|---|---|
| HTTPValidationError | [HTTPValidationError](../accelbyte_py_sdk/api/amalgam_game_telemetry/models/http_validation_error.py) |
| TelemetryBody | [TelemetryBody](../accelbyte_py_sdk/api/amalgam_game_telemetry/models/telemetry_body.py) |
| ValidationError | [ValidationError](../accelbyte_py_sdk/api/amalgam_game_telemetry/models/validation_error.py) |
